### ISSUE TITLE
 Icongenie  ERROR: incorrect resolution!  when using --padding

### ISSUE THREAD

**Issue by diadal** [quasarframework/quasar#7101]
icongenie verify failed when using padding option for example --padding 10,10  

Xcode Building fail 

```
 Mode SPA 

 ERROR: incorrect resolution! - png:          src/statics/icons/favicon-128x128.png 
 ERROR: incorrect resolution! - png:          src/statics/icons/favicon-96x96.png 
 ERROR: incorrect resolution! - png:          src/statics/icons/favicon-32x32.png 
 ERROR: incorrect resolution! - png:          src/statics/icons/favicon-16x16.png 
 SIZE OK - ico:          src/statics/icons/favicon.ico 

 Mode CAPACITOR 

 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-hdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-hdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-hdpi/ic_launcher.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-mdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-mdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-mdpi/ic_launcher.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xhdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xhdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xhdpi/ic_launcher.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxhdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxhdpi/ic_launcher.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-mdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-mdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-hdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-hdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-xhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-xhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-xxhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-xxhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-xxxhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-xxxhdpi/splash.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@1x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@2x-1.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@3x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-29x29@1x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-29x29@2x-1.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-29x29@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-29x29@3x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-40x40@1x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-40x40@2x-1.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-40x40@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-40x40@3x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-60x60@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-60x60@3x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-76x76@1x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-76x76@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-83.5x83.5@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-512@2x.png 
 SIZE OK - splashscreen: src-capacitor/ios/App/App/Assets.xcassets/Splash.imageset/splash-2732x2732-1.png 
 SIZE OK - splashscreen: src-capacitor/ios/App/App/Assets.xcassets/Splash.imageset/splash-2732x2732-2.png 
 SIZE OK - splashscreen: src-capacitor/ios/App/App/Assets.xcassets/Splash.imageset/splash-2732x2732.png 
```

**Comment by pdanpdan** [comment]
Can you please check the size for `src/statics/icons/favicon-128x128.png`?

**Comment by diadal** [comment]
the --padding value added to size if you --padding 10,10 size will be 138 by 138 if --padding 10,5
size will be be 138 by 133


<img width="399" alt="Screenshot 2020-05-25 at 12 02 13 PM" src="https://user-images.githubusercontent.com/19895516/82807475-bbfa4e00-9e7f-11ea-9309-09a387afc651.png">

**Comment by anubhavkamath** [comment]
Having the same issue. I'm not able to add the padding. If I do, the app icon becomes much smaller (the build seems to resize it).

**Comment by rstoenescu** [comment]
Thank you for reporting!
Fix will be available in "@quasar/icongenie" v2.2.1.

**Comment by diadal** [comment]
issue not yet resolved 

 `Running @quasar/icongenie v2.2.0`

 Mode SPA 

 ERROR: incorrect resolution! - png:          src/statics/icons/favicon-128x128.png 
 ERROR: incorrect resolution! - png:          src/statics/icons/favicon-96x96.png 
 ERROR: incorrect resolution! - png:          src/statics/icons/favicon-32x32.png 
 ERROR: incorrect resolution! - png:          src/statics/icons/favicon-16x16.png 
 SIZE OK - ico:          src/statics/icons/favicon.ico 

 Mode CAPACITOR 

 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-hdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-hdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-hdpi/ic_launcher.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-mdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-mdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-mdpi/ic_launcher.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xhdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xhdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xhdpi/ic_launcher.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxhdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxhdpi/ic_launcher.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_foreground.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png 
 ERROR: incorrect resolution! - png:          src-capacitor/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-mdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-mdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-hdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-hdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-xhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-xhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-xxhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-xxhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-land-xxxhdpi/splash.png 
 SIZE OK - splashscreen: src-capacitor/android/app/src/main/res/drawable-port-xxxhdpi/splash.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@1x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@2x-1.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@3x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-29x29@1x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-29x29@2x-1.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-29x29@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-29x29@3x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-40x40@1x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-40x40@2x-1.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-40x40@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-40x40@3x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-60x60@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-60x60@3x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-76x76@1x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-76x76@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-83.5x83.5@2x.png 
 ERROR: incorrect resolution! - png:          src-capacitor/ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-512@2x.png 
 SIZE OK - splashscreen: src-capacitor/ios/App/App/Assets.xcassets/Splash.imageset/splash-2732x2732-1.png 
 SIZE OK - splashscreen: src-capacitor/ios/App/App/Assets.xcassets/Splash.imageset/splash-2732x2732-2.png 
 SIZE OK - splashscreen: src-capacitor/ios/App/App/Assets.xcassets/Splash.imageset/splash-2732x2732.png

**Comment by rstoenescu** [comment]
Haven’t I mentioned that it’s fixed in icongenie 2.2.1? @diadal
